packages/core/src/utilities/grid/_grid-col.scss

```diff
@@ -1,3 +1,5 @@
+@import 'styles/tools/index';
```

Comment:
I'm wondering if `tools` would be better-named `mixins`?

Reply:
Yeah I could try this. I might even combine mixins with the `settings` folder so that you have 1 import for all variables, mixins, and other stuff you might include in the future like placeholders or functions

Reply:
I like the idea of adding it to settings actually even better 🎉